### ISSUE TITLE
Add accessibility attributes to Tabs component elements

### ISSUE THREAD

**Issue by jeanduplessis** [sourcegraph/sourcegraph#15403]
## Problem

The Tabs component is missing some aria attributes to make it fully accessible.

## Requirements

Add the following aria attributes to the relevant elements of the Tabs component:
- aria-selected="true/false" on the tab
- aria-controls="#id" on the tab to reference the tabpanel
- aria-labelledby="#id" on the tabpanel to reference the tab

Reference implementation: https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

## Prior art

Initial accessibility attributes were added in this PR: https://github.com/sourcegraph/sourcegraph/pull/15391
